src/js/helpers/util.js

```diff
@@ -13,6 +13,61 @@ var util = {
     /*eslint-disable no-alert */
     return global.prompt.apply(null, arguments);
     /*eslint-enable no-alert */
+  },
+  param: function (obj) {
+    try {
+      return Object.keys(obj).reduce(function (a, k) {
+        a.push(k + "=" + encodeURIComponent(obj[k]));
+        return a;
+      }, []).join("&");
+    } catch (e) {
+      return obj;
+    }
+  },
+  serializeArray: function (form) {
```

Comment:
@rcorral did you see [the utils tests](https://github.com/mesosphere/marathon-ui/blob/1972ffccb6191bc2e7066933a6b2ae511221071e/src/test/util.test.js)? I think they qualify :)